apps/openmw/mwmechanics/aiwander.hpp

```diff
@@ -151,6 +151,8 @@ namespace MWMechanics
             static const std::string sIdleSelectToGroupName[GroupIndex_MaxIdle - GroupIndex_MinIdle + 1];
 
             static int OffsetToPreventOvercrowding();
+
+            float fIdleChanceMultiplier;
```

Comment:
Is there a reason to make it a member variable and not use the old implementation with a static variable in the function?